src/classes/CW_BatchAudienceGroupRelationAssignTest.cls

```diff
@@ -0,0 +1,62 @@
+/**
+* File Name   :    CW_BatchAudienceGroupRelationAssignTest.cls
+* Description :    B-115203 - Target and Control group Naming Convention
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 13/05/2014 Alex Panka     Initial version
+* 0.2 10/07/2014 Amar Reddy     
+*
+*/
+@IsTest
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.